examples/stm32/f3/nucleo-f303re/Makefile.include

```diff
@@ -0,0 +1,34 @@
+##
+## This file is part of the libopencm3 project.
+##
+## Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
+## Copyright (C) 2010 Piotr Esden-Tempski <piotr@esden.net>
+## Copyright (C) 2011 Fergus Noble <fergusnoble@gmail.com>
+## Copyright (C) 2016 Josh Myer <josh@joshisanerd.com>
```

Comment:
I don't personally believe there's anything copyrightable by uwe, piotr or fergus in this file.